main.tf

```diff
@@ -126,6 +127,9 @@ resource "google_container_cluster" "k8s_cluster" {
     horizontal_pod_autoscaling {
       disabled = ! var.enable_addon_horizontal_pod_autoscaling
     }
+    dns_cache_config {
```

Comment:
you can add a link to `NodeLocal DNSCache` here - 
https://cloud.google.com/kubernetes-engine/docs/how-to/nodelocal-dns-cache

Reply:
I have added this in the Variable's description. Shouldn't that be enough?